test/system/projects/members_test.rb

```diff
@@ -411,31 +411,31 @@ def setup
       visit namespace_project_members_url(@namespace, @project)
 
       assert_text 'Displaying 5 items'
-      assert_selector '#project-members table tbody tr', count: 5
-      assert_selector "#project-members table tbody tr td:nth-child(#{username_col})", text: @member_james.user.email
-      assert_selector "#project-members table tbody tr td:nth-child(#{username_col})", text: @member_jean.user.email
-      assert_selector "#project-members table tbody tr td:nth-child(#{username_col})", text: @member_joan.user.email
-      assert_selector "#project-members table tbody tr td:nth-child(#{username_col})", text: @member_john.user.email
-      assert_selector "#project-members table tbody tr td:nth-child(#{username_col})", text: @member_ryan.user.email
+      assert_selector 'table tbody tr', count: 5
```

Comment:
**nitpick:** I'm not too fond of removing the id from these tests.  The reason is that if a second table was added anywhere on this page then the test wouldn't be specific and it could lead to flaky tests.

Reply:
As discussed, we'll leave this as is since the id was part of an issue with tab_component id being improperly placed